src/System.Management.Automation/engine/ErrorPackage.cs

```diff
@@ -184,6 +185,117 @@ public enum ErrorCategory
         NotEnabled = 31,
     }
 
+    /// <summary>
+    /// Defines the <see cref="SuggestionInfo" /> class.
+    /// </summary>
+    public class SuggestionInfo
+    {
+        /// <summary>
+        /// Gets the string representation of the suggestion object.
+        /// </summary>
+        /// <param name="suggestion">The suggestion to convert to string.</param>
+        public static implicit operator string(SuggestionInfo suggestion) => suggestion.ToString();
+
+        internal static readonly Dictionary<string, SuggestionInfo> MiscSuggestionLibrary =
+            new Dictionary<string, SuggestionInfo>
+            {
+                {
+                    "Transaction",
+                    new SuggestionInfo(
+                        ScriptBlock.CreateDelayParsedScriptBlock(
+                            SuggestionStrings.Suggestion_StartTransaction, isProductCode: true))
```

Comment:
```suggestion
                            SuggestionStrings.Suggestion_StartTransaction, isProductCode: true
                        )
                    )
```

Reply:
Personally, _I_ certainly prefer this format, but it's not a style generally used in this repo. 😄